media-plugins/swh-lv2/swh-lv2-1.0.16.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="Large collection of LV2 audio plugins/effects"
+HOMEPAGE="http://plugin.org.uk/"
+SRC_URI="https://github.com/swh/lv2/archive/v${PV}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="amd64 x86"
+IUSE=""
+
+RDEPEND=">=sci-libs/fftw-3.0.0"
+DEPEND="${RDEPEND}
+	dev-libs/libxslt
+	virtual/pkgconfig"
+
+S="${WORKDIR}/lv2-${PV}"
+
```

Comment:
I see things like that in the compile log:
```
cc -Wall -I. -Iinclude  -fPIC -DPIC -DFFTW3 -march=skylake -mtune=skylake -O2 -pipe -ggdb3  util/blo.c -c -o util/blo.o
```

So I guess tc-getCC was still needed